docs/guides/dataloaders.md

```diff
@@ -74,6 +74,14 @@ await loader.load(1)
 
 Will result in only one call to `load_users`.
 
+To load multiple values in a single call we can use the `load_many`
+function, which has the same interface as `load_fn`, but uses the
+the DataLoader cache.
```

Comment:
What do you mean by "has the same interface"? Do you mean the same signature? If so, they're slightly different (one takes a `List[]` and the other an `Iterable[]`.

Reply:
Yeah, I meant signature. Anyway I tweaked the wording.